tutor/hooks/catalog.py

```diff
@@ -57,6 +57,11 @@ def run_this_on_start(root, config, name):
     #: :parameter str name: docker-compose project name.
     COMPOSE_PROJECT_STARTED: Action[[str, Config, str]] = Action()
 
+    #: Called whenever the config is loaded. This action is called at the end of the save command.
+    #:
+    #: :parameter dict config: project configuration.
```

Comment:
Please specify that modifying this object will not trigger changes in the configuration. For all purposes, it should be considered read-only.